### PR TITLE
add Cheats command

### PR DESCRIPTION
**Pull request by AuraOfTheDawn**

### CODE REVIEW

File: src/responses/citra.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
...
       "log": { "reply": "This forum topic tells you how to __get the log file__: <https://community.citra-emu.org/t/how-to-upload-the-log-file/296>"},
       "canary": { "reply": "The nightly build of Citra contains already reviewed and tested features. If you require support with the installation or use of Citra, or you want to report bugs you should use this version.\nThe Canary build of Citra is the same as our nightly builds, with additional features that are still waiting on review before making it into the official Citra builds. We will not provide support for issues found only in this version. If you believe you’ve found a bug, please retest on our nightly builds. Both versions are still in development, so expect crashes and bugs."},
+      "cheats": { "reply": "Looking to add cheats to Citra? Drop a .txt file with the Title ID of the game you're playing into the Cheats folder. You can find it by clicking 'File -> Open Citra Folder'. \n Cheats are stored in the gateway format. A sample collection of cheats are located at <https://github.com/JourneyOver/CTRPF-AR-CHEAT-CODES/tree/master/Cheats/> "},
```
Comment (zhaowenlan1779) [inline src/responses/citra.json]: extra spaces around `\n`

Comment (AuraOfTheDawn) [reply src/responses/citra.json]: Removed the space. Should anything else get changed?

Comment (zhaowenlan1779) [reply src/responses/citra.json]: If I understand correctly you needn't the space before `\n` either.

**Comment by valentinvanelslande** [comment]
nissue
\n isn't displayed correctly

**Comment by FearlessTobi** [comment]
The PR isn't merged yet and the message got sent manually. @valentinvanelslande

**Pull request merged**